vpn/src/main/java/com/duckduckgo/mobile/android/vpn/ui/onboarding/VpnStore.kt

```diff
@@ -42,11 +43,14 @@ interface VpnStore {
 }
 
 @ContributesBinding(AppScope::class)
+@SingleInstanceIn(AppScope::class)
```

Comment:
We don't really need this but I thought it would save some migrations. No strong opinions, I can remove it